app/models/user.js

```diff
@@ -29,6 +29,26 @@ module.exports = (sequelize, DataTypes) => {
         values: [REGULAR_ROLE, ADMIN_ROLE],
         allowNull: false,
         defaultValue: 'regular'
+      },
+      position: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        defaultValue: 0,
+        get() {
+          const rating = this.getDataValue('position');
+          if (rating >= 50) {
```

Comment:
I think you could use a switch case here

Reply:
ok